### PR TITLE
templates: add template for requesting a new platform

### PR DESCRIPTION
**Pull request by bgilbert**
Requestors are unlikely to have all this information up front, but it'd be good to document what information we need and have a centralized place to collect it.

cc @arithx @darkmuggle @dustymabe @jlebon @lucab

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-platform.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- [ ] How can the OS retrieve network configuration?  Is DHCP sufficient, or is there some other network-accessible metadata service?
+
+- [ ] In particular, how can the OS retrieve the system hostname?
+
+- [ ] Does the platform require the OS to have a specific console configuration?
+
+- [ ] Is there a mechanism for the OS to report to the platform that it has successfully booted?  Is the mechanism required?
+
+- [ ] Does the platform have an agent that runs inside the instance?  Is it required?  What does it do?
```
Comment (arithx) [inline .github/ISSUE_TEMPLATE/new-platform.md:28]: Probably worth asking for links to source code if available & what language it's written in.

**Review by arithx — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/new-platform.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+name: Request a new platform
+about: Ask for Fedora CoreOS to support a new cloud environment
+title: 'Platform Request: <platform name>'
+labels: 'kind/enhancement'
```
Comment (jlebon) [inline .github/ISSUE_TEMPLATE/new-platform.md:5]: Minor: `areas/platforms` would also be appropriate I think

**Review by jlebon — APPROVED** [review]
This looks really nice!

A follow-up to this would be to have a checklist for actually implementing support for it (e.g. see checklist in https://github.com/coreos/fedora-coreos-tracker/issues/355#issue-556377201). Depending on what's actually required though some of those checkboxes might not apply (e.g. if the platform reuses an existing artifact, and we just need to upload it to the cloud).

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by bgilbert** [comment]
@jlebon Good thought.  Filed https://github.com/coreos/fedora-coreos-tracker/issues/675 as a follow-up issue.

**Pull request merged**